doc/README.rst

```diff
@@ -0,0 +1,255 @@
+  account both using console and ssh client due to for security reasons sudo is
+  configured for asking the password.
+
+Another reason to require the support password and encrypt it is to make easy
+enforcing full control and audit of who have admin access to each VM. This is
+because the SSH public is not specific of each VM, but the password yes. There
+are alternatives but they are more complex. For example, do not provide the SSH
+private key to the support staff but use an special designed ssh-agent that audit
+each access. This document describes the `protocol used by OpenSSH's ssh-agent`_.
+
+Top_
+
+
+Why each region must use their own public keys
+==============================================
+
+The region staff team are responsible of the virtual machines instantiated on their
+servers. Therefore each organisation should have the control of who access the virtual
+machines for support purposes and set and enforce the corresponding policy. It is
+not possible if the public keys are shared among all the organisations. Additionally,
+it is also extremely insecure and a problem when an organisation left the federation.
+
+Top_
+
+
+How the keys are injected inside the virtual machines. Limitations
+==================================================================
+
+The support account is created by a script invoked at boot time. This script is named
+*activate_support_account.py* and it is available at `GlanceSync support scripts`_.
+Inside this folder there is also documentation and all the material to create the base
+images with the support account.
+
+The first thing that the script has to do is to check if a support account is not already
+created and the UUID of the virtual machine has not changed; in these cases the script
+end after printing by console the encrypted password. To encrypt this password we use
+the public GPG key provided by the FIWARE Lab administrator node. To decrypt the password
+you have to use your private GPG key, we have developed an specific script that can be
+found in section `Obtaining the password of the support account`_.
+
+The checking of the UUID is done because a template could be created from a running virtual
+machine when the support account is already created; the support account should be deleted
+before doing the snapshot, otherwise the password of the support account will be the same
+in all the images instantiated from that template.
+
+The next step, the script tries to download the userdata file from the metadata server.
+This is something that cloudinit also do, but this script can do more attempts because
+it works in the background. However, if the metadata server is not accessible
+(e.g. because the network is down) the script will finally give up. In this case, it will
+use as failback a public SSH and public GPG keys hardcoded in the images. The failback
+keys are used also if the user modify the userdata content and removes the part that
+provides the keys.
```

Comment:
We must say that the 'hardcoded' keys used by the script if this case occurs are only managed by the _FIWARE Support Team_. Then, admins of other nodes will have to request support to this team in order to get the password of the root o get access to that VMs.